src/test/crimson/seastore/transaction_manager_test_state.h

```diff
@@ -0,0 +1,85 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#pragma once
+
+#include <random>
+
+#include "test/crimson/gtest_seastar.h"
```

Comment:
Is it possible not to depend on gtest_seastar.h? It would be great if `TMTestState` could be used by non-test tools. See https://github.com/ceph/ceph/blob/7a0c6c5d97c0a13865c04ba182560cd1d285771b/src/tools/crimson/perf_staged_fltree.cc#L17